### PR TITLE
Rentals controller

### PR DESCRIPTION
**Pull request by pataus91**
- Creation of rentals controller
- Fixed rentals routes (index is not nested to bikes)
- Created basics views (no front-end)
@etienneburdet

### CODE REVIEW

File: app/controllers/rentals_controller.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class RentalsController < ApplicationController
+  def index
+    @rentals = Rental.all
+  end
+
+  def new
+    @bike = Bike.find(params[:bike_id])
```
Comment (etienneburdet) [inline app/controllers/rentals_controller.rb]: Make set_bike method

File: app/controllers/rentals_controller.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    @rental = Rental.new
+  end
+
+  def create
+    @rental = Rental.new(rental_params)
+    @bike = Bike.find(params[:bike_id])
```
Comment (etienneburdet) [inline app/controllers/rentals_controller.rb]: use set_bike method

File: app/controllers/rentals_controller.rb (hunk 1 above)
Comment (etienneburdet) [reply app/controllers/rentals_controller.rb]: no need for @bike

File: app/controllers/rentals_controller.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
+class RentalsController < ApplicationController
+  def index
```
Comment (etienneburdet) [inline app/controllers/rentals_controller.rb]: Add : before_action set_bike, only: [] .

File: config/routes.rb [hunk 4]
```diff
@@ -2,4 +2,9 @@
   devise_for :users
   root to: 'pages#home'
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+  resources :bikes, only: [ :index, :destroy, :update, :create, :new] do
+    resources :rentals, only: [ :new, :create ]
+  end
+  resources :rentals, only: [ :index ]
```
Comment (etienneburdet) [inline config/routes.rb]: Add private method set_bike

**Comment by pataus91** [comment]
@etienneburdet code modified. I think ready to merge after your check

File: app/controllers/rentals_controller.rb (hunk 3 above)
Comment (etienneburdet) [reply app/controllers/rentals_controller.rb]: pas besoin pour new

**Pull request merged**